### PR TITLE
Обновление библиотеки bem-components

### PR DESCRIPTION
**Pull request by mishanga**
TODO: spin

### CODE REVIEW

**Comment by tadatuta** [comment]
:+1:

File: desktop.blocks/comment/comment.bemtree [hunk 1]
```diff
@@ -160,7 +160,7 @@ block('comment')(
                                 {
                                     block: 'input',
                                     mods: {
-                                        theme: 'normal',
+                                        theme: 'islands',
                                         type: 'textarea',
```
Comment (sipayRT) [inline desktop.blocks/comment/comment.bemtree]: why not `block : 'textarea'`? As I know, there is no such type for `input`.

Comment (mishanga) [inline desktop.blocks/comment/comment.bemtree]: WIP ;)

Comment (sipayRT) [inline desktop.blocks/comment/comment.bemtree]: sorry :)

**Comment by tavriaforever** [comment]
Update, need review 
// cc @tadatuta

File: bower.json [hunk 2]
```diff
@@ -23,13 +23,13 @@
   ],
   "dependencies": {
     "bem-core": "git://github.com/bem/bem-core.git#v2.5.0",
-    "bem-components": "git://github.com/bem/bem-components.git#b2e0a63522f0636da40996eb6d4042c09f48e1ff",
+    "bem-components": "git://github.com/bem/bem-components.git#34773736897c67868d8962e382b9a2b9d0558f28",
```
Comment (tadatuta) [inline bower.json]: why not `d559759ab99a6321b833d0d6ab369ddea4ab96db`?

File: bower.json [hunk 3]
```diff
@@ -23,13 +23,13 @@
   ],
   "dependencies": {
     "bem-core": "git://github.com/bem/bem-core.git#v2.5.0",
```
Comment (tadatuta) [inline bower.json]: you don't need to specify `bem-core` explicitly as it'll be provided by `bem-components` anyway

File: bower.json [hunk 4]
```diff
@@ -23,13 +23,13 @@
...
-    "bem-components": "git://github.com/bem/bem-components.git#b2e0a63522f0636da40996eb6d4042c09f48e1ff",
+    "bem-components": "git://github.com/bem/bem-components.git#34773736897c67868d8962e382b9a2b9d0558f28",
     "bem-history": "git://github.com/bem/bem-history.git#v2",
     "bem-content": "git://github.com/tavriaforever/bem-content.git#747cae3c15a15a56a6d8eaedbee4674b6ddec21c"
```
Comment (tadatuta) [inline bower.json:27]: I think it's better to send PR to @verybigman

File: bower.json [hunk 5]
```diff
@@ -23,13 +23,13 @@
...
   },
   "resolutions": {
     "bem-core": "v2.5.0",
```
Comment (tadatuta) [inline bower.json]: not sure if all the libs are needed in `resolutions`

File: desktop.blocks/comment/comment.bemtree [hunk 6]
```diff
@@ -155,40 +155,34 @@ block('comment')(
...
-                                        type: 'textarea',
-                                        size: 'l',
-                                        'has-clear' : true
+                            block: 'forum-create',
+                            content: {
+                                block: 'textarea',
+                                mods: { theme: 'islands', size: 'l', width: 'available' },
+                                name: 'body',
+                                val: this.ctx.body,
+                                placeholder: 'Ответ',
```
Comment (tadatuta) [inline desktop.blocks/comment/comment.bemtree]: i18n?

File: desktop.blocks/forum-create/forum-create.bemhtml [hunk 7]
```diff
@@ -7,14 +7,16 @@ block('forum-create')(
...
-                        mods: { theme: 'normal', size: 's', type: 'button' },
+                        mods: { theme: 'islands', size: 's', type: 'button' },
+                        val: 'source',
                         options: [
                             {
+                                type: 'button',
                                 val: 'source',
-                                icon: { block: 'icon', mods: { view: 'markdown', size: 24 }, title: 'Маркдаун' },
-                                checked: true
+                                icon: { block: 'icon', mods: { view: 'markdown', size: 24 }, title: 'Маркдаун' }
```
Comment (tadatuta) [inline desktop.blocks/forum-create/forum-create.bemhtml]: i18n or even better use just English here

File: desktop.blocks/forum-header/_view/forum-header_view_auth-no.bemtree [hunk 8]
```diff
@@ -8,8 +8,8 @@ block('forum-header').mod('view', 'auth-no')(
             {
                 block: 'button',
                 url: req.forumUrl + 'auth/',
-                text: BEM.I18N('forum-header', 'login') + ' (github.com)',
-                mods: { theme: 'normal', size: 'm', view: 'action', type: 'link' },
+                text : BEM.I18N('forum-header', 'login') + ' (github.com)',
```
Comment (tadatuta) [inline desktop.blocks/forum-header/_view/forum-header_view_auth-no.bemtree]: strange spaces

File: desktop.blocks/forum-create/__view/forum-create__view.styl [hunk 9]
```diff
@@ -1,6 +1,7 @@
 .forum-create .forum-create__view
 {
     min-height: 150px;
+    height: 150px;
     margin-top: 10px;
     width: 100%;
     display: none;
```
Comment (gela-d) [inline desktop.blocks/forum-create/__view/forum-create__view.styl:3]: let's write properties by groups :)

```
display: none;

width: 100%;
height: 150px;
...
```

File: bower.json (hunk 2 above)
Comment (tavriaforever) [inline bower.json]: will update

File: bower.json (hunk 4 above)
Comment (tavriaforever) [inline bower.json:27]: If you remember, we discussed about it and agreed that we must do this in another way

File: bower.json (hunk 3 above)
Comment (tavriaforever) [inline bower.json]: ok

File: desktop.blocks/comment/comment.bemtree (hunk 6 above)
Comment (tavriaforever) [inline desktop.blocks/comment/comment.bemtree]: yes, I will add

File: desktop.blocks/forum-create/forum-create.bemhtml (hunk 7 above)
Comment (tavriaforever) [inline desktop.blocks/forum-create/forum-create.bemhtml]: I will add an i18n

File: desktop.blocks/forum-header/_view/forum-header_view_auth-no.bemtree (hunk 8 above)
Comment (tavriaforever) [inline desktop.blocks/forum-header/_view/forum-header_view_auth-no.bemtree]: agree

File: bower.json (hunk 5 above)
Comment (tavriaforever) [inline bower.json]: I do not agree

> Unable to find a suitable version for bem-core, please choose one:
>     1) bem-core#~2.2.4 which resolved to 2.2.4 and is required by bem-history#fb308cad79
>     2) bem-core#2.5.0 which resolved to 2.5.0 and is required by bem-components#d559759ab9, bem-components#d559759ab9
>     3) bem-core#v2.5.0 which resolved to 2.5.0 and is required by bem-content#747cae3c15Prefix the choice with ! to persist it to bower.json

**Comment by tavriaforever** [comment]
Update PR

**Comment by tadatuta** [comment]
lgtm

**Pull request merged**